src/lib/components/NeedlePlot.react.js

```diff
@@ -528,15 +525,6 @@ NeedlePlot.propTypes = {
      * properties change
      */
     setProps: PropTypes.func,
-
-    /**
-     *
-     */
-    dashEvents: PropTypes.oneOf(['click', 'hover', 'unhover', 'selected']),
```

Comment:
So these were listed as available events but were never actually emitted?

Reply:
@alexcjohnson Yeah, this code was copied from somewhere (like example code for another React component for Dash) and `dashEvents` and `fireEvent` were just never removed from the code.